lib/eldap/src/eldap.erl

```diff
@@ -12,30 +12,74 @@ References:
 
 The above publications can be found at [IETF](http://www.ietf.org).
 
-## DATA TYPES
+Terminology abbreviations:
 
-Type definitions that are used more than once in this module:
+- Dn. An LDAPDN is defined to be the representation of a Distinguished Name
+   (DN) after encoding according to the specification in [RFC4514].
+-
```

Comment:
This dash looks strange on the generated page.

Reply:
Thank you for your time and comment. Let me fix it.